src/pages/orgs/[slug]/home/urgeButton.component.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { Button, Popconfirm, Skeleton } from 'antd';
+import { ThunderboltFilled } from '@ant-design/icons';
+import { Link } from '@tidb-community/ui';
+
+import * as Styled from './home.styled';
+
+const TooltipContent = () => (
+  <Styled.Tooltips>
+    <b>加急贴响应时效</b>
+    <br />
+    工作时间：周一至周五 10:00 - 18:00 一小时内响应
+    <br />
+    非工作时间（含法定节假日）：顺延至下一个工作日 11:00 前响应（如遇线上紧急事故，请
+    <Link href="https://tidb.io/contact-us/incident" target="_blank" rel="noreferrer">
```

Comment:
`contact-us` belongs to the same NextJS application, do we need to include `host name` in the URL?